specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/examples/ComposeModels.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "body": {},
+    "composeRequest": {
+      "ModelsIds": [ "ModelA", "ModelB" ]
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {
+        "Location": "{endpoint}/formrecognizer/v2.0-preview/custom/models/{modelId}"
```

Comment:
"v2.0-preview", should we put {version}  or {v2.1-preview.1} here?

Reply:
Done, thank you